README.md

```diff
@@ -1,12 +1,10 @@
 # react-a11y-announcer
 
-Welcome to React Announcer from Think Company.
```

Comment:
I updated the line to include our name, I think it is important to have everything in one sentence up front for SEO, and to make it ad rem.